CMakeLists.txt

```diff
@@ -1,69 +1,62 @@
 cmake_minimum_required(VERSION 2.6)
+
 if(${CMAKE_VERSION} STREQUAL "2.8.2")
-  # bug http://vtk.org/Bug/view.php?id=11020
-  message( WARNING "CMake/CPack version 2.8.2 will not create working .deb packages!")
-endif(${CMAKE_VERSION} STREQUAL "2.8.2")
+	# Bug http://vtk.org/Bug/view.php?id=11020
+	message(WARNING "CMake/CPack version 2.8.2 will not create working .deb packages!")
+endif()
 
 # This can be read from ${PROJECT_NAME} after project() is called
-project(minetest)
+project(Minetest)
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
 
-set(VERSION_EXTRA "" CACHE STRING "Stuff to append to version string")
 
-# Also remember to set PROTOCOL_VERSION in clientserver.h when releasing
+# Also remember to set PROTOCOL_VERSION in network/networkprotocol.h when releasing
 set(VERSION_MAJOR 0)
 set(VERSION_MINOR 4)
 set(VERSION_PATCH 12)
-set(VERSION_PATCH_ORIG ${VERSION_PATCH})
+set(VERSION_EXTRA "" CACHE STRING "Stuff to append to version string")
 
+set(VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
 if(VERSION_EXTRA)
-	set(VERSION_PATCH ${VERSION_PATCH}-${VERSION_EXTRA})
+	set(VERSION_STRING ${VERSION_STRING}-${VERSION_EXTRA})
 else()
 	# Comment the following line during release
-	set(VERSION_PATCH ${VERSION_PATCH}-dev)
+	set(VERSION_STRING "${VERSION_STRING}-dev")
 endif()
-set(VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
 
-MESSAGE(STATUS "*** Will build version ${VERSION_STRING} ***")
+message(STATUS "*** Will build version ${VERSION_STRING} ***")
 
-# Configuration options
 
+# Configuration options
+set(DEFAULT_RUN_IN_PLACE FALSE)
 if(WIN32)
-	set(RUN_IN_PLACE 1 CACHE BOOL "Run directly in source directory structure")
-else()
-	set(RUN_IN_PLACE 0 CACHE BOOL "Run directly in source directory structure")
+	set(DEFAULT_RUN_IN_PLACE TRUE)
 endif()
+set(RUN_IN_PLACE ${DEFAULT_RUN_IN_PLACE} CACHE BOOL
+	"Run directly in source directory structure")
```

Comment:
This way the rest of the command (such as the documentation string) isn't duplicated.